internal/services/compute/shared_image_version_resource_test.go

```diff
@@ -120,6 +120,157 @@ func TestAccSharedImageVersion_blobURI(t *testing.T) {
+			Config:  r.setup(data),
+			Destroy: false,
+			Check: acceptance.ComposeTestCheckFunc(
+				data.CheckWithClientForResource(ImageResource{}.virtualMachineExists, "azurerm_virtual_machine.testsource"),
+				data.CheckWithClientForResource(ImageResource{}.generalizeVirtualMachine(data), "azurerm_virtual_machine.testsource"),
+			),
+		},
+		{
+			// First create a shared image version with initial signature Template Name
+			Config: r.imageVersionUefiSettingsTemplates(data, signatureTemplateNames),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		// The security profile and hence the uefi_settings are not returned by the API
+		// so we need to ignore them
+		data.ImportStep(
+			"uefi_settings.#",
+			"uefi_settings.0.%",
+			"uefi_settings.0.signature_template_names.#",
+			"uefi_settings.0.signature_template_names.0",
+			"uefi_settings.0.additional_signatures.#",
+		),
+		{
+			// Then add additional signature template name (forces a new resource)
+			Config: r.imageVersionUefiSettingsTemplates(data, updatedSignatureTemplateNames),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		// The security profile and hence the uefi_settings are not returned by the API
+		// so we need to ignore them
+		data.ImportStep(
+			"uefi_settings.#",
+			"uefi_settings.0.%",
+			"uefi_settings.0.signature_template_names.#",
+			"uefi_settings.0.signature_template_names.0",
+			"uefi_settings.0.signature_template_names.1",
+			"uefi_settings.0.additional_signatures.#",
+		),
+	})
+}
+
+func TestAccSharedImageVersion_uefiSettingsDb(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_shared_image_version", "test")
+	r := SharedImageVersionResource{}
+
+	signatureTemplateNames := `["MicrosoftUefiCertificateAuthorityTemplate"]`
+	certDb := "MIIDNzCCAh+gAwIBAgIRANcuAK10JUqNpehWlkldzxEwDQYJKoZIhvcNAQELBQAwFzEVMBMGA1UEAxMMQ3VzdG9tRGJLZXkzMB4XDTIzMDYxOTEwNTI0MloXDTMzMDYxNjEwNTI0MlowFzEVMBMGA1UEAxMMQ3VzdG9tRGJLZXkzMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAq+QdB6n3TDk12Qa/JcbmdfEpIrx4dKG9d5D/SRHWfJACInxtH64jzvGohVnAqIDqcKK+FvVLDPrqD7hbzV34AOXkyVoRtHEsdDErkG9CVBJlWleuew+if9TkW8wabFT3/sHSzVbG6+6AFOHsnDbO1Rpvh1ZPp2AgqiNg7XUHQM9zH00BYz7xtL9XEr+sRRgp0Bn0PGQGQU1Q302TK6jlHwJGMidke4Le2IIDJTUTGx3yWuX7f/T/u6alZeKjg+hYysJ7dpaaC5DyRTT5pJv62pZBJa3DkwWWSKroJozp9ujf93KYP7NoCLHkyiITAUK04hsHm/UvIt7ZhayTS24MbwIDAQABo34wfDAfBgNVHSMEGDAWgBQBXPUO5tTx8gh9G1iwS1KMwXUi/zAVBglghkgBhvhCAQEBAf8EBQMDAPABMBMGA1UdJQQMMAoGCCsGAQUFBwMDMA4GA1UdDwEB/wQEAwIEsDAdBgNVHQ4EFgQUAVz1DubU8fIIfRtYsEtSjMF1Iv8wDQYJKoZIhvcNAQELBQADggEBAA4xZmr3HhDOc2xzDMjqiVnCBMPT8nS9P+jCXezTeG1SIWrMmQUSs8rtU0YoNRIq1wbT/rqbYIwwhRfth0nUGf22zp4UdigVcpt+FQj9eGgeF6sJyHVWmMZu8rEi8BhHEsS6jHqExckp0vshhyW5whr86znWFWf/EsVGFkxd7kjv/KB0ff2ide5vLOWxoTfYmSxYyg2K1FQXP7L87Rb7O6PKzo0twVgeZ616e/yFLcmUQgnHBhb2IKtdo+CdTCxcw9/nNqGPwsNLsti2jyr5oNm9mX6wVaAuXCC3maX35DdWFVK0gXcENEw+Q6+JSyPV1ItXc5CD0NU9pd+R85qsFlY="
```

Comment:
for consistency with other tests, it would be nice to move this to a separate testdata file